src/character.h

```diff
@@ -1653,6 +1653,7 @@ class Character : public Creature, public visitable
         int get_mod_stat_from_bionic( const character_stat &Stat ) const;
         // route for overmap-scale traveling
         std::vector<tripoint_abs_omt> omt_path;
+        std::vector<tripoint_abs_omt> revealed_omts;
```

Comment:
I'm not sure this is desirable, #70480 for example makes that map(visitor's pass) reveal in a 300-OMT-radius around the player. Since it's going to be tagging *every single road*, if we assume roads are like... 25% of the map, that 90,000 OMT search range results in 22,500 entries. There is an open PR for a new map that searches in 24x24OM range (would result in an astounding 4.665m tripoints having to be serialized, just for roads!!!).

That said I have not tried this in practice.

Reply:
I don't think roads are 25% of the map (that would require every four overmap tiles to contain a road which is clearly too dense even for cities). Still, should performance become a concern, then this vector probably need to be refactored into something stored along with the overmap, because 4.665M tripoints are going to take a while to store considering the reallocation of `std::vector`.

Reply:
That would be a huge vector to serialize, but luckily the map in that other PR doesn’t reveal any roads so it won’t be quite that bad.

Reply:
I don't think we need to worry about serialization in this PR either which way. If someone wants to tackle it later, with any potential performance concerns: Fine. Good, even.

But for the purposes of this PR, characters forgetting because you saved, quit, and reloaded is fine.